akka-docs-dev/rst/scala/stream-flows-and-basics.rst

```diff
@@ -117,39 +117,39 @@ specification, which Akka is a founding member of.
+Streams, guarantees that it will never emit more elements than the received total demand for any given ``Subscriber``.
 
 .. note::
-   The Reactive Streams specification defines its protocol in terms of **Publishers** and **Subscribers**.
-   These types are *not* meant to be user facing API, instead they serve as the low level building blocks for
+   The Reactive Streams specification defines its protocol in terms of ``Publishers`` and ``Subscriber``s.
+   These types are **not** meant to be user facing API, instead they serve as the low level building blocks for
    different Reactive Streams implementations.
 
-   Akka Streams implements these concepts as **Sources**, **Flows** (referred to as **Processor** in Reactive Streams)
-   and **Sinks** without exposing the Reactive Streams interfaces directly.
+   Akka Streams implements these concepts as ``Source``s, ``Flow``s (referred to as ``Processor`` in Reactive Streams)
+   and ``Sink``s without exposing the Reactive Streams interfaces directly.
    If you need to integrate with other Reactive Stream libraries read :ref:`reactive-streams-integration-scala`.
 
 The mode in which Reactive Streams back-pressure works can be colloquially described as "dynamic push / pull mode",
-since it will switch between push or pull based back-pressure models depending on if the downstream is able to cope
-with the upstreams production rate or not.
+since it will switch between push and pull based back-pressure models depending on the downstream being able to cope
+with the upstream production rate or not.
 
-To illustrate further let us consider both problem situations and how the back-pressure protocol handles them:
+To illustrate this further let us consider both problem situations and how the back-pressure protocol handles them:
 
 Slow Publisher, fast Subscriber
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-This is the happy case of course–we do not need to slow down the Publisher in this case. However signalling rates are
+This is the happy case of course – we do not need to slow down the Publisher in this case. However signalling rates are
 rarely constant and could change at any point in time, suddenly ending up in a situation where the Subscriber is now
 slower than the Publisher. In order to safeguard from these situations, the back-pressure protocol must still be enabled
 during such situations, however we do not want to pay a high penalty for this safety net being enabled.
 
 The Reactive Streams protocol solves this by asynchronously signalling from the Subscriber to the Publisher
-`Request(n:Int)` signals. The protocol guarantees that the Publisher will never signal *more* than the demand it was
-signalled. Since the Subscriber however is currently faster, it will be signalling these Request messages at a higher
+``Request(n: Int)`` signals. The protocol guarantees that the Publisher will never signal *more* elements than the
```

Comment:
I can't, because I don't have the doc-generation-machinery. What we had before – single backquotes – didn't do any formatting. I was assuming that we need double backquotes like in numerous other places. So what's wrong here? Is it the whitespace I introduced?